<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionDataView</class>
 <widget class="QWidget" name="SessionDataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_gameInfo">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidget_gameInfo">
          <column>
           <property name="text">
            <string>Key</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_stageIDs">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget_stageIDs">
          <column>
           <property name="text">
            <string>Stage ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_fighterIDs">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget_fighterIDs">
          <column>
           <property name="text">
            <string>Fighter ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_baseStatusIDs">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="tableWidget_baseStatusIDs">
          <column>
           <property name="text">
            <string>Status ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Enum Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Short Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Custom Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_specificStatusIDs">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTableWidget" name="tableWidget_specificStatusIDs">
          <column>
           <property name="text">
            <string>Fighter ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Enum Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Short Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Custom Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_hitStatusIDs">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableWidget" name="tableWidget_hitStatusIDs">
          <column>
           <property name="text">
            <string>Hit Status ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Enum Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_playerData">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QStackedWidget" name="stackedWidget_playerData"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
